src/main/java/com/google/dicomwebfuse/fuse/DicomFuse.java

```diff
@@ -65,7 +65,7 @@ public int getattr(String path, FileStat fileStat) {
       dicomFuseHelper.checkExistingObject(dicomPath);
       dicomFuseHelper.setAttr(dicomPath, this, fileStat);
     } catch (DicomFuseException e) {
-      LOGGER.debug(e);
+      LOGGER.error("getattr error!", e);
```

Comment:
Will this result in the permission denied error message being sent to stdout instead of the debug file?

Also, can we include the full HTTP error message instead of just the status code?

Reply:
> Also, can we include the full HTTP error message instead of just the status code?

Done
> Will this result in the permission denied error message being sent to stdout instead of the debug file?

I've changed the current PR. I've added AccessChecker with a detailed error message for permission denied error. If a user makes a typo in --datasetAddr parameter, or project or DICOM Store does not exist, or role for account service key is not DICOM Editor, will be presented following error in stdout:
![error](https://user-images.githubusercontent.com/39695947/62046963-72c37880-b233-11e9-8f6f-1831f3bb5eae.png)